services/github/github-package-lock-json.tester.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+
+const { ServiceTester } = require('../tester')
+const { semverRange } = require('../validators')
+
+const t = (module.exports = new ServiceTester({
+  id: 'GithubPackageLockJson',
+  title: 'GithubPackageLockJson',
+  pathPrefix: '/github/package-lock-json',
+}))
+
+t.create('Dependency version')
+  .get('/dependency-version/paulmelnikow/react-boxplot/react.json')
+  .expectBadge({
+    label: 'react',
+    message: semverRange,
+  })
+
+t.create('Dependency version - Custom label')
+  .get(
+    '/dependency-version/paulmelnikow/react-boxplot/react.json?label=react%20tested'
+  )
+  .expectBadge({
+    label: 'react tested',
+    message: semverRange,
+  })
+
+t.create('Dependency version')
+  .get('/dependency-version/paulmelnikow/react-boxplot/simple-statistics.json')
+  .expectBadge({
+    label: 'simple-statistics',
+    message: semverRange,
+  })
+
```

Comment:
Lets get rid of these two tests as they are unnecessary. The label specification is common across any and every service and is well tested elsewhere already. The second test with `simple-statistics` is duplicative of the one above with `react`
```suggestion
```